addons/project/controllers/portal.py

```diff
@@ -414,7 +414,12 @@ def _prepare_tasks_values(self, page, date_begin, date_end, sortby, search, sear
             domain = AND([domain, self._task_get_search_domain(search_in, search, milestones_allowed, project)])
 
         # content according to pager and archive selected
-        group_field = None if groupby == 'none' else groupby
+        if groupby == 'none':
+            group_field = None
+        elif groupby == 'priority':
+            group_field = 'priority desc'
+        else:
+            group_field = groupby
```

Comment:
Change the key in `_task_get_searchbar_groupby` instead

Reply:
> Change the key in `_task_get_searchbar_groupby` instead

Yesterday, we discussed this on Discord. We cannot pass it as a key because it is used in the group by, which causes a traceback error.
` grouped_tasks = [Task_sudo.concat(*g) for k, g in groupbyelem(tasks, itemgetter('priority desc'))]  `